travis-ymls/ant-tar-patched.travis.yml

```diff
@@ -0,0 +1,20 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : ant-tar-patched
+# Source Repo         : https://github.com/Zimbra/ant-tar-patched
+# Travis Job Link     : https://travis-ci.com/github/nageshlop/ant-tar-patched/builds/213433084
+# Created travis.yml  : Yes
+# Maintainer          : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Script License      : Apache 2.0
+#
+# ----------------------------------------------------------------------------
+dist: xenial
+language: java
+arch:
+  - amd64
+  - ppc64le
+before_install:
+  - sudo apt-get install ant
+script: 
+  - ant -f /home/travis/build/nageshlop/ant-tar-patched/build.xml
```

Comment:
Hard coded path. Try to replace it with travis env variables. I think `TRAVIS_BUILD_DIR` should help.

More details: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

Reply:
Sure will update.